app/views/unattended/provisioning_templates/registration/linux_registration_default.erb

```diff
@@ -23,6 +23,8 @@ set -e
 <%= snippet 'remote_execution_ssh_keys' %>
 <%= snippet 'insights' -%>
 
+<%= install_packages(host_param('host_registration_packages')) unless host_param('host_registration_packages').blank? %>
```

Comment:
should we rename this to `host_init_packages` due to the upcoming rename of the template?

Reply:
I'm not sure, all the other parameters are starting with `host_registration_*`,
I would say the relation to the registration process is more than to the template kind.

Reply:
I think we said we want to use the same mechanism also in provisioning. Also for other params https://projects.theforeman.org/issues/32115

The "host init" seems to be applicable both in registration and provisioning processes